### ISSUE TITLE
Display next image based on folder order

### ISSUE THREAD

**Issue by MildPanda** [d2phap/ImageGlass#457]
Unsure exactly how to phrase this so I'll just say how it currently seems to work VS what I'd want it to do

**What it does:** When you press left or right arrow it appears display the next file in alphabetical order

**What I'd like:** If you tell the folder to organize any way (file name, resolution, size, etc) it'll display the next file in that order so when you click on the specific image(s) you want to look at pressing left or right arrow will still provide image(s) you wanted to view. I don't know if this is possible but if it is I'd love to see this implemented into imageglass, thanks!

In short, show images by folder structure.

Here is a video demonstrating this https://youtu.be/L3XPNKfqk3A

**Comment by d2phap** [comment]
Hi @MildPanda 
Thanks for the issue, however it is a duplication of #139 
The team has already worked around for this issue.
